### PR TITLE
st394 - Add Session languages

### PR DESCRIPTION
**Pull request by gtryus**
Card: https://trello.com/c/i6PPaPJX

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/664)

### CODE REVIEW

File: SIL.Archiving/Extensions.cs [hunk 1]
```diff
@@ -112,13 +112,17 @@ public static void AppendDelimiter(this StringBuilder sb, string value, string d
 		public static string ToLatinOnly(this string s, string charReplacement, string spaceReplacement, string charsToIgnore)
```
Comment (jasonleenaylor) [inline SIL.Archiving/Extensions.cs]: You could adjust the comment on the method to mention the single '.' feature in <remarks>

File: SIL.Archiving/IMDI/Schema/IMDI_3_0.cs [hunk 2]
```diff
@@ -1993,7 +2128,7 @@ public string Task
 			}
 			set
 			{
-				MDGroup.Content.SubGenre = value.ToVocabularyType(false, ListType.Link(ListType.ContentSubGenre));
+				MDGroup.Content.Task = value.ToVocabularyType(false, ListType.Link(ListType.ContentSubGenre));
```
Comment (jasonleenaylor) [inline SIL.Archiving/IMDI/Schema/IMDI_3_0.cs]: Well, what an interesting bug that was.

**Comment by gtryus** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[SIL.Archiving/IMDI/Schema/IMDI_3_0.cs, line 2131 at r1](https://beta.reviewable.io/reviews/sillsdev/libpalaso/664#-L8401rC_uqEVuijP8A9-r1-2131:-L8E-3MN6IUfWEgPwgTi:bvyf6ix) ([raw file](https://github.com/sillsdev/libpalaso/blob/38c9d999f8059bb575beb6ca31091796ce0f309e/SIL.Archiving/IMDI/Schema/IMDI_3_0.cs#L2131)):*
<details><summary><i>Previously, jasonleenaylor (Jason Naylor) wrote…</i></summary><blockquote>

Well, what an interesting bug that was.
</blockquote></details>

Yes, the symptom was you would set the subgenre but it would be exported as unspecified. Tricky to find because this code isn't at all related. It looks like the bug would have come from copying code but not making all necessary changes. (I broke off the Extensions changes because they actually touched different files and didn't have to be lumped with everything else.) I was hoping I could get them approved more easily.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/sillsdev/libpalaso/664)*

**Comment by gtryus** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[SIL.Archiving/Extensions.cs, line 112 at r1](https://beta.reviewable.io/reviews/sillsdev/libpalaso/664#-L8401rC_uqEVuijP8A5-r1-112:-L8E-f9a8RX-Flk7RctW:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/38c9d999f8059bb575beb6ca31091796ce0f309e/SIL.Archiving/Extensions.cs#L112)):*
<details><summary><i>Previously, jasonleenaylor (Jason Naylor) wrote…</i></summary><blockquote>

You could adjust the comment on the method to mention the single '.' feature in <remarks>
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/sillsdev/libpalaso/664)*

**Comment by gtryus** [comment]
I will put up a new pull request once we get Alex input. He may want to use a different strategy to solve this problem.

**Pull request closed without merging**